src/command_modules/azure-cli-monitor/setup.py

```diff
@@ -12,7 +12,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "0.0.14"
+VERSION = "0.0.15"
```

Comment:
If this is a breaking change, should it not be 0.1.0?

Reply:
I would tend to agree though it's still 0.x..

Reply:
ok.